internal/checks/common/assets/consts.rego

```diff
@@ -4,6 +4,7 @@ details := details {
 	details := {
 		"organization_missing_minimal_permissions": "Organization is missing minimal permissions",
 		"repository_missing_minimal_permissions": "Repository is missing minimal permissions",
+		"repository_data_is_missing": "Repository is not fetched",
```

Comment:
can you please re-order those strings by groups based: not fetched, missing permission, others